docs/field_types.md

```diff
@@ -183,46 +184,6 @@ class BooleanField(Field):
         return '1' if value else '0'
 ```
 
-Here's another example - a field for storing UUIDs in the database as 16-byte strings. We'll use Python's built-in `UUID` class to handle the conversion from strings, ints and tuples into UUID instances. So in our Python code we'll have the convenience of working with UUID objects, but they will be stored in the database as efficiently as possible:
```

Comment:
this example still may be valid but with additional comment for a Clickhouse version where UUID is not available

Reply:
right